.cicd/generate-pipeline.sh

```diff
@@ -483,7 +483,8 @@ EOF
 EOF
         fi
         if [[ "$ROUND" != "$ROUNDS" ]]; then
-            echo '  - wait'
+            echo '  - wait: ~'
+            echo '    continue_on_failure: true'
             echo ''
         fi
```

Comment:
From [AUTO-1236](https://blockone.atlassian.net/browse/AUTO-1236) (emphasis mine):
> This story expands the functionality of the pipeline code to support this by setting `continue_on_failure: true` on the `wait` steps in the auto-generated `pipeline.yml` for **_scheduled builds_** in the [eosio-test-stability](https://buildkite.com/EOSIO/eosio-test-stability) pipeline.
>
> **_It is imperative this flag is never set in [...] manual eosio-test-stability runs_**, or any other EOSIO pipelines as this will waste resources and cause false-negative (type 2) error with respect to build outcomes and pull request checks.

In this case, the `continue_on_failure` flag will be set for all test stability runs, including manual ones. I think something like this would work:
```bash
if [[ "$ROUND" != "$ROUNDS" ]]; then
    echo '  - wait'
    [[ "$BUILDKITE_SOURCE" == 'schedule' ]] && echo '    continue_on_failure: true'
    echo ''
fi
```
Alternatively, you could provide an override for anyone who _does_ want manual builds to run all steps regardless of outcome.
```bash
if [[ "$ROUND" != "$ROUNDS" ]]; then
    echo '  - wait'
    [[ "$BUILDKITE_SOURCE" == 'schedule' || "$CONTINUE_ON_FAILURE" == 'true' ]] && echo '    continue_on_failure: true'
    echo ''
fi
```
I certainly do not think `continue_on_failure` should be the default because the manual runs are only used to prove stability. A single failure shows instability, so if we exit right away then this shortens test times by hours and saves a bunch of unnecessary cloud spend.

Reply:
If you really want to follow the ticket to the letter, you could do something like this...
```bash
if [[ "$ROUND" != "$ROUNDS" ]]; then
    echo '  - wait'
    [[ "$BUILDKITE_SOURCE" == 'schedule' && "$BUILDKITE_PIPELINE_SLUG" == 'eosio-test-stability' ]] && echo '    continue_on_failure: true'
    echo ''
fi
```
...but I no longer believe that is necessary because anyone doing multiple rounds in a scheduled build on another EOSIO pipeline is doing it intentionally and knows what they are doing.

Reply:
I think the second one is probably better.